test/xpu_api/utilities/optional/optional.specalg/make_optional_explicit.pass.cpp

```diff
@@ -0,0 +1,55 @@
+// -*- C++ -*-
```

Comment:
Can we merge `make_optional_explicit.pass.cpp` and `make_optional_explicit_initializer_list.pass.cpp` with `make_optional.pass.cpp`?

Reply:
Test with name `make_optional.pass.cpp` already exist.
We should have some unique name here.

Reply:
I take a look at these tests and I think better to save them as is, in 3 separate files.
They check different things...

Reply:
Fixed